.github/workflows/prod-release.yaml

```diff
@@ -26,122 +26,101 @@ jobs:
+          with:
+            source_image: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller"
+            image_tag: "${{ github.event.inputs.tag }}"
+            region: "me-south-1"
+            repo_host: "${{ secrets.PROD_BAH_AWS_ACCOUNT }}.dkr.ecr.me-south-1.amazonaws.com"
+            role: "${{ secrets.PROD_BAH_AWS_ROLE }}"
+
+      - name: Deploy Images to CPT
+          uses: ./.github/actions/push-image
+          with:
+            source_image: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller"
+            image_tag: "${{ github.event.inputs.tag }}"
+            region: "af-south-1"
+            repo_host: "${{ secrets.PROD_CPT_AWS_ACCOUNT }}.dkr.ecr.af-south-1.amazonaws.com"
+            role: "${{ secrets.PROD_CPT_AWS_ROLE }}"
+
+      - name: Deploy Images to HKG
+          uses: ./.github/actions/push-image
+          with:
+            source_image: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller"
+            image_tag: "${{ github.event.inputs.tag }}"
+            region: "ap-east-1"
+            repo_host: "${{ secrets.PROD_HKG_AWS_ACCOUNT }}.dkr.ecr.ap-east-1.amazonaws.com"
+            role: "${{ secrets.PROD_HKG_AWS_ROLE }}"
+
+      - name: Deploy Images to MXP
+          uses: ./.github/actions/push-image
+          with:
+            source_image: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller"
+            image_tag: "${{ github.event.inputs.tag }}"
+            region: "eu-south-1"
+            repo_host: "${{ secrets.PROD_MXP_AWS_ACCOUNT }}.dkr.ecr.eu-south-1.amazonaws.com"
+            role: "${{ secrets.PROD_MXP_AWS_ROLE }}"
+
+      - name: Deploy Images to CGK
+          uses: ./.github/actions/push-image
+          with:
+            source_image: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller"
+            image_tag: "${{ github.event.inputs.tag }}"
+            region: "ap-southeast-3"
+            repo_host: "${{ secrets.PROD_CGK_AWS_ACCOUNT }}.dkr.ecr.ap-southeast-3.amazonaws.com"
+            role: "${{ secrets.PROD_CGK_AWS_ROLE }}"
+
+      - name: Deploy Images to BJS
+          uses: ./.github/actions/push-image
+          with:
+            source_image: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller"
+            image_tag: "${{ github.event.inputs.tag }}"
+            region: "cn-north-1"
+            repo_host: "${{ secrets.PROD_BJS_AWS_ACCOUNT }}.dkr.ecr.cn-north-1.amazonaws.com.cn"
+            role: "${{ secrets.PROD_BJS_AWS_ROLE }}"
+
+      - name: Deploy Images to ZHY
+          uses: ./.github/actions/push-image
+          with:
+            source_image: "${{ secrets.BETA_AWS_ACCOUNT }}.dkr.ecr.us-west-2.amazonaws.com/amazon/appmesh-controller"
+            image_tag: "${{ github.event.inputs.tag }}"
+            region: "cn-north-1"
```

Comment:
I accidentally stumbled upon this PR. Shouldn't this value be `cn-northwest-1`? 🤔